helpers.R

```diff
@@ -1313,3 +1313,48 @@ dt_ranges_overlap <- function(range1, range2){
 
     return(ranges_overlap)
 }
+
+read_combine_ws_traits <- function(ws_prod, ws_var, new_var_name, dmns, 
```

Comment:
where is this used? we don't have ws_traits in the portal data anymore, but we can bring them back if you need them for something.